idaes/generic_models/properties/core/generic/generic_property.py

```diff
@@ -1263,9 +1263,17 @@ def initialize(blk, state_args=None, state_vars_fixed=False,
                             blk[k].true_to_appr_species[p, j])
                     # Need to calculate all flows before doing mole fractions
                     for p, j in blk[k].params.apparent_phase_component_set:
-                        calculate_variable_from_constraint(
-                            blk[k].mole_frac_phase_comp_apparent[p, j],
-                            blk[k].appr_mole_frac_constraint[p, j])
+                        x = value(
+                            blk[k].flow_mol_phase_comp_apparent[p, j] /
+                            sum(blk[k].flow_mol_phase_comp_apparent[p, jj]
+                                for jj in blk[k].params.apparent_species_set))
+                        lb = blk[k].mole_frac_phase_comp_apparent[p, j]._lb
```

Comment:
Why are you directly accessing the `_lb` attribute?  The public `lb` attribute returns a float or `None` (the `_lb` attribute could be an expression...)